django/contrib/contenttypes/management/__init__.py

```diff
@@ -43,7 +40,40 @@ def rename_backward(self, apps, schema_editor):
         self._rename(apps, schema_editor, self.new_model, self.old_model)
 
 
-def inject_rename_contenttypes_operations(
+class MoveContentType(migrations.RunPython):
```

Comment:
As far as I'm aware, we should document this new operations (probably [here](https://docs.djangoproject.com/en/stable/ref/contrib/contenttypes/)).